wagtail/admin/views/pages.py

```diff
@@ -792,7 +792,7 @@ def move_confirm(request, page_to_move_id, destination_id):
 
 
 def set_page_position(request, page_to_move_id):
-    page_to_move = get_object_or_404(Page, id=page_to_move_id)
+    page_to_move = get_object_or_404(Page, id=page_to_move_id).specific
```

Comment:
Why do we need `specific` here?

Reply:
Part of the problem the client is having is that they are subscribing to the `post_save` signal using their specific page types as the `sender`, with the expectation that moving will result in receipt of the signal.

While `Page.move()` does in-turn call `Page.save()`, resulting in a `post_save` signal being emitted, because we are only dealing with a vanilla `Page` object here, that signal is only emitted with `Page` as the sender, and not the fully-fledged page type.

Reply:
The current behaviour is inconsistent with the `move_page()` view just above this, which does ultimately result in a `post_save` signal being emitted for the specific page type.

Reply:
@tomdyson / @kaedroho Have removed this change for now to simplify reviewing